webapp/src/mutator.ts

```diff
@@ -975,12 +975,13 @@ class Mutator {
         fromTemplate = false,
         description = 'duplicate card',
         asTemplate = false,
+        propertiesThatMeetFilters?: Record<string, string> | undefined,
```

Comment:
Let's rename this to something like `propertyOverrides`. That name has context in `kanban.tsx`, but not here.

Reply:
Done.